src/main/resources/org/gradle/profiler/report/report-template.html

```diff
@@ -0,0 +1,324 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Benchmark Results</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.5.2/css/bootstrap.min.css" integrity="sha512-MoRNloxbStBcD8z3M/2BmnT+rg4IsMxPkXaGh2zD6LGNNFE80W3onsAhRcMAMrSoyWL9xD7Ert0men7vR8LUZg==" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/select2/4.0.13/css/select2.min.css">
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.5.2/js/bootstrap.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/select2/4.0.13/js/select2.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/handlebars.js/4.7.6/handlebars.min.js"></script>
+    <script src='https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.9.3/Chart.bundle.min.js'></script>
+    <style type="text/css">
+#data-table {
+    overflow-x: scroll;
+}
+th {
+    font-size: larger;
+    text-align: center;
+}
+td.numeric {
+    text-align: right;
+    font-family: monospace;
+}
+td.diff {
+    font-size: 9pt; color: #c0c0c0;
+}
+td.WARM_UP {
+    background-color: #fff8ee;
+}
+tr:hover td.WARM_UP {
+    background-color: #ffe6d4;
+}
+td.MEASURE {
+    background-color: #eeffee;
+}
+tr:hover td.MEASURE {
+    background-color: #d8ffd8;
+}
+div.controls > div.col-form-label {
+    text-align: right;
+}
+    </style>
+    <script>
+const Phase = {
+    WARM_UP: "WARM_UP",
+    MEASURE: "MEASURE"
+};
+
+let benchmarkResult =
+@@BENCHMARK_RESULT_JSON@@
+;
+</script>
+    <script id="data-table-template" type="text/x-handlebars-template">
```

Comment:
Is there an advantage in using the `script` tag for the template instead of a `template` tag?

Reply:
This was recommended by [Handlebars' docs](https://handlebarsjs.com/installation/#usage):

> **Always use a script-tag for your template**
> 
> If you use this method, you have to wrap your template with a script-tag. Otherwise the browser may remove or modify parts of your template if you don't. Have a look at ["Unexpected markup in tables"](https://html.spec.whatwg.org/multipage/parsing.html#unexpected-markup-in-tables) for an example.

I tried using `<template>` but it didn't work.

Reply:
👍